webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -551,6 +568,16 @@ def _get_selected_ensembles_from_store(
             ]
         return None
 
+    def _format_index_value(axis: pd.Index, index: int) -> Union[int, datetime.date]:
```

Comment:
I think that it would be more consistent with the other functions if we had them all after `controller`. What do you think?

Reply:
yes sorry, of course! done.